clang/test/Driver/sycl-spirv-ext-win.c

```diff
@@ -0,0 +1,108 @@
+// REQUIRES: system-linux
```

Comment:
now that `sycl-spirv-ext.c` is run for Windows - this new test is probably not needed (plus the fact that it still has 'system-linux' on a -win.c test)

Reply:
Ah. I intended to NOT add that file. Hazards of 'git add .'.
Will remove it.

Thanks